docs/release-notes/2.31.0.md

```diff
@@ -0,0 +1,166 @@
+
+- [PMM-10495](https://jira.percona.com/browse/PMM-10495): By default, Swagger UI attempts to validate specs against swagger.io’s online validator. For PMM 2.31.0, we have disabled it to safeguard your privacy.
+
+
+- [PMM-10064](https://jira.percona.com/browse/PMM-10064): Introduced UI changes on the PMM Inventory pages for better readability and user experience.
+
+
+
+- [PMM-10076](https://jira.percona.com/browse/PMM-10076): For an improved UI experience, we have enforced the enter button for the date picker, which performs the same action as clicking `apply time range`.
+
+- [PMM-7806](https://jira.percona.com/browse/PMM-7806): Upgraded postgres_exporter version used in pmm from 0.8.0 to 0.10.1. 
+
+  
+- [PMM-10018](https://jira.percona.com/browse/PMM-10018): Lowered the default values of CPU and memory for haproxy while deploying a DBaaS [Percona XtraDB Cluster](https://www.percona.com/software/mysql-database/percona-xtradb-cluster) (PXC) cluster to avoid wasting resources.
+
+
+    
+- [PMM-9118](https://jira.percona.com/browse/PMM-9118): Refined the error message thrown when a secure connection cannot be established while adding a monitoring database to PMM, thus making it much more contextual and easy to troubleshoot.
+
+
+- [PMM-10516](https://jira.percona.com/browse/PMM-10516): 
+
+- [PMM-10095](https://jira.percona.com/browse/PMM-10095): While removing the service with pmm-admin, now you do not have to specify the service name for a single service. PMM will automatically look up this service and remove it.
+
+- [PMM-9989](https://jira.percona.com/browse/PMM-9989): PMM now supports Percona Operator for MongoDB 1.12.0.
+
+
+
+## Bugs Fixed
+
+- [PMM-10628](https://jira.percona.com/browse/PMM-10628): While adding MongoDB or PostgreSQL database via socket, PMM was throwing an error. This has been fixed in PMM 2.31.0.
+
+- [PMM-10624](https://jira.percona.com/browse/PMM-10624): Empty dataname label after PostgreSQL upgraded in PMM.
+
+- [PMM-10608](https://jira.percona.com/browse/PMM-10608): Fixed issues pertaining to the new Home Dashboard, such as high CPU panel unreadable for a higher number of nodes (more than ten nodes), filtering for the environment as well as node name not working correctly, time range not working as expected, some panels on the dashboard not displaying data for selected filters, and so on.
+
+- [PMM-10587](https://jira.percona.com/browse/PMM-10587):Fixed an issue where if you hover over the telemetry icon stacktrace with error appears.
+
+- [PMM-10513](https://jira.percona.com/browse/PMM-10513): Fixed an issue where pmm-admin summary stored PMM password in client/status.json, thus posing a security risk.
+
+- [PMM-10065](https://jira.percona.com/browse/PMM-10065): Fixed an issue encountered while monitoring a large number of nodes (1000) and services. Here, the changing scrape frequency led to data not being captured on the dashboard and throwing a templating error.
```

Comment:
@anton-bystrov have we fixed it completely or only in a new dashboard?

Reply:
we have adjusted some VM defaults that would prevent it for some large number of monitoring services. But also probably still depends how many services you have :)

Original issue was fixed.

Reply:
@BupycHuk resolve this comment?